RELEASE-NOTES

```diff
@@ -11,7 +11,7 @@ This release includes the following changes:
  o NPN: remove support for and use of [16]
  o Websockets: initial support [23]
 
-This release includes the following bugfixes:
+This release includes the following bug fixes:
```

Comment:
We use the contracted word bugfix all over this project and the spellchecker accepts that.

We also have scripts and infra that checks for that word, so changing it would require more changes.

Reply:
Fixed